apisix/plugins/ext-plugin/init.lua

```diff
@@ -611,6 +611,14 @@ local rpc_handlers = {
                 end
             end
 
+            local len = rewrite:RespHeadersLength()
```

Comment:
Should add some test cases.

Also, shall we exclude some vital headers such as `Connection`, `Content-Length`, `Transfer-Encoding` and etc.

Reply:
OK,I'll add some test cases.  
And exclude some important headers like Connection,Content-Length,Transfer-Encoding, Location,Server, WWW-Authenticate, Content-Encoding, Content-Type,Content-Location,Content-Type,Content-Language, feel free to give any other suggesions.

Reply:
Let's see others' suggestions about this.

Reply:
> Connection,Content-Length,Transfer-Encoding, Location,Server, WWW-Authenticate, Content-Encoding, Content-Type,Content-Location,Content-Type,Content-Language

LGTM